src/index.js

```diff
@@ -51,7 +51,7 @@ const _responses = {
     500: "Unexpected error"
 };
 
-function _request(path, method, init) {
+function _request(path, method, init=undefined, credentialResolver=undefined, retry=true) {
```

Comment:
Please use named parameters and destructuring in situations like this, like:

```
function _request(path, method, {init, credentialResolver, retry }) {
```

or:

```
function _request(path, method, options) {
  const {init, credentialResolver, retry } = options
 /* ...  */
}
```

It makes calls like:

```
return _request(`${impactUrl}/api`, "GET",  undefined, undefined, false).then(apiInfo => {
```

more easily readable:

```
return _request(`${impactUrl}/api`, "GET",  { retry: false }).then(apiInfo => {
```

Reply:
I went with Magnus approach where there is no option as input to the function.